### PR TITLE
Added routes for login

### PR DESCRIPTION
**Pull request by vladpetrenciuc**
Login doesn't work, and still have to do the routes for trackController

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 dotenv.config();
-//mongodb connection
-var mongoose = require('mongoose');
-mongoose.connect('mongodb+srv://' + process.env.MONGOUSER + ':' + process.env.MONGOPASSWORD + '@cluster0.jlrkv.mongodb.net/'+process.env.MONGODBNAME+'?retryWrites=true&w=majority', { useNewUrlParser: true })
+
+//passport
+var passport = require('passport');
+app.use(passport.initialize());
+app.use(passport.session());
+passport = require('./config/passport');
```
Comment (Radu94) [inline app.js]: you're overwriting the passport var from line 13 with this require. Did you mean to require the passport module from `./config/passport` and use that in your app ? If so just do 
```javascript
const passport = require('./config/passport')
app.use(passport.initialize());
app.use(passport.session());
```

**Review by Radu94 — CHANGES_REQUESTED** [review]
Try and replace `var` with `let` & `const`: 
* `let` for variables that change values
* `const` for variables that don't 

It will help you avoid reinitialisation errors

File: app.js (hunk 1 above)
Comment (vladpetrenciuc) [reply app.js]: For some reason I thought that I first need to run initialize and session methods from the passport module, and then overwrite it. I've corrected it.

Comment (vladpetrenciuc) [reply app.js]: As a clarification, this correction did not solve the login problem. I've omitted to mention this, sorry

Comment (Radu94) [reply app.js]: please push changes so we can take another look :)

File: routes/routes.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const passport = require('passport');
```
Comment (Radu94) [inline routes/routes.js]: here you're using a different `passport` instance then the one you configured in `config/passport`

Comment (vladpetrenciuc) [reply routes/routes.js]: I replaced it and I get "TypeError: passport.use is not a function"

File: app.js [hunk 3]
```diff
@@ -3,12 +3,6 @@ var expressLayouts = require('express-ejs-layouts');
 var session=require('express-session');
```
Comment (Radu94) [inline app.js]: use `const` or `let` instead of `var`

**Review by Radu94 — CHANGES_REQUESTED** [review]

**Review by Radu94 — APPROVED** [review]

**Pull request merged**